index.html

```diff
@@ -19,21 +20,171 @@
+                    <button type="button">
+                        See Project
+                    </button>
+                </div>
+                <div class="card">
+                    <img class="preview" src="/images/SnapshootPortfolio4.png" alt="Website Preview">
+                    <h2 >
+                        Multi-Post Stories
+                    </h2>
+                    <div class="type_and_date">
+                        <span>CANOPY</span>
+                        <span>●</span>
+                        <span>Front End Dev</span>
+                        <span>●</span>
+                        <span>2023</span>
+                    </div>
+                    <p>
+                        A daily selection of privately 
+                        personalized reads; no accounts or
+                        sign-ups required.
+                    </p>
+                    <ul class="prog_lang">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <button type="button">
+                        See Project
+                    </button>
+                </div>
+            </section>
+                     
+                <section class="aboutme">
+                    <h2>About Myself</h2>
+                    <p> 
+                        Hello I'm a software developer! I can help you 
+                        build a product, feature or website Look 
+                        through some of my work and 
+                        experience! If you like what you see and
+                        have a project you need coded, don't 
+                        hesitate to contact me. 
+                    </p>
+                    <h3>
+                        <a class="connect_link" href="">LET'S CONNECT</a>
+                    </h3>
+                    <ul class="icons">
+                        <li><a href="https://twitter.com/MohametAlmeari"><i class="fa fa-twitter"></i></a></li>
+                        <li><a href="https://www.linkedin.com/in/mohamet-almeari"><i class="fa fa-linkedin"></i></a></li>
+                        <li><a href="https://github.com/mohametalmeari"><i class="fa fa-github"></i></a></li>
+                        <li><a href=""><i class="fa fa-instagram"></i></a></li>
+                    </ul>
+
+                    <button type="button">
+                        Get my resume
+                    </button>
+                    <ul class="footer_list">
+                        <li>
+                            Languages <i class="fa fa-angle-right"></i>
+                        </li>
+                        <li>
+                            Frameworks <i class="fa fa-angle-right"></i>
+                        </li>
+                        <li>
+                            Skills <i class="fa fa-angle-right"></i>
+                        </li>
+                    </ul>
```

Comment:
- [optional] Please kindly pay good attention to the tech stacks, your design is not in accordance to the given template. ℹ️ 

## Your design
<img width="291" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227506980-97bfcee0-ab50-4ac7-992b-73dead92afa8.png">

## Figma design
<img width="145" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227507191-bba852ef-9f5d-4b28-a9cf-5026536208e3.png">